### PR TITLE
upload data to dish detail

### PR DESCRIPTION
**Pull request by jolietran-se**

### CODE REVIEW

**Comment by jolietran-se** [comment]
Ready

File: resources/views/dishes/show.blade.php [hunk 1]
```diff
@@ -1,51 +1,146 @@
...
+                                        </div>
+                                        <div class="product-info__review">
+                                            <div class="rating2">
+                                                <a>{{ $dish->like_number }} <span class="fa fa-heart rating-text"></span></a>
+                                            </div>
+
+                                        </div>
+                                        <div class="divider divider--xs product-info__divider hidden-xs"></div>
+                                        <div class="product-info__description hidden-xs">
+                                            <div class="product-info__description__brand"><img src="{{ asset('images/avatars/2.jpg') }}" class="avatar_chef" alt=""/></div>
```
Comment (BunnyPi04) [inline resources/views/dishes/show.blade.php]: base link đưa vào config

**Comment by BlazingRockStorm** [comment]
Reject

**Comment by phuongyeol** [comment]
> Reject

Why why why :))

**Comment by jolietran-se** [comment]
Ready

**Comment by BlazingRockStorm** [comment]
REJECTTTTTT

**Pull request closed without merging**